sparklingpandas/__init__.py

```diff
@@ -33,8 +33,14 @@
 
 if 'IS_TEST' not in os.environ:
     VERSION = '0.0.4'
-    JAR_FILE = 'sparklingpandas_2.10-' + VERSION + '-SNAPSHOT.jar'
-    jar = os.path.join(sys.prefix, 'jar', JAR_FILE)
+    JAR_FILE = 'sparklingpandas-assembly-' + VERSION + '-SNAPSHOT.jar'
+    my_location = os.path.dirname(os.path.realpath(__file__))
+    prefixes = [os.path.join(my_location, '../target/scala-2.10/'),
+                os.path.join(my_location, '../curret-release/'),
+                os.path.join(sys.prefix, "local/jars/")]
+    jars = map(lambda prefix: os.path.join(prefix, JAR_FILE), prefixes)
+    print jars
+    jar = filter(lambda path: os.path.exists(path), jars)[0]
```

Comment:
filter is deprecated, you can use list comprehensions with a filtering clause instead.